src/go/rpk/pkg/kafka/client.go

```diff
@@ -208,33 +207,6 @@ func configureTLS(
 		tlsConf = &tls.Config{RootCAs: caCertPool}
 		log.Debug("API TLS enabled using rpk.tls")
 
-	// Enable client auth if the cert & key files are set for the redpanda
```

Comment:
Can you add an explanation for this part, please.

Reply:
Sure. Rpk first checks for TLS config in the `rpk.tls` field in the config file. Otherwise, it checked for `redpanda.kafka_api_tls`. However, since it's a list now, you can't rely on it anymore.

Reply:
So what functionality have we lost?

Reply:
Now, to be able to use TLS on rpk you *have* to set `rpk.tls`.

Reply:
Maybe we could iterate the endpoints until we find one that resolves and connects successfully, or add a flag that takes the name of the named connection.

Reply:
I don't think it's s blocker, but raise a ticket to improve the UX.